pkg/registry/apis/dashboard/legacy/query_dashboards.sql

```diff
@@ -44,7 +46,7 @@ WHERE dashboard.is_folder = {{ .Arg .Query.GetFolders }}
     {{ end }}
     {{ if .Query.GetTrash }}
     AND dashboard.deleted IS NOT NULL
-    {{ else if .Query.LastID }}
+    {{ else }}
```

Comment:
What's the reasoning for this change?

Reply:
I enabled `dashboardRestore = true ;` and then noticed that deleted dashbaords show up in our normal listing.... 

So i fixed it 😄